static/css/restyle/restyle.less

```diff
@@ -2050,3 +2050,51 @@ body.editor-tools {
   }
 
 }
+
+/* Source/Diff viewer */
+body.file-viewer {
+  pre, code, kbd, tt, samp, tt {
+    font-family: "andale mono", monospace !important;
```

Comment:
Not using `!important` for the primary font-family was a concious hack to avoid needing to overriding all the other places different fonts were in use. I think this use of `!important` is OK, but of course this does highlight why such solutions aren't great.